app/controllers/sufia/resource_sync_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Sufia::ResourceSyncController < ApplicationController
+  def source_description
+    host = request.host
+    # Caching based on host, for multitenancy support
+    body = Rails.cache.fetch("source_description_#{host}", expires_in: 1.week) do
```

Comment:
There's a reason we don't write all the code on one line (we could), it's so that it's more readable.